tests/integration/studies_blueprint/test_disk_usage.py

```diff
@@ -78,3 +82,19 @@ def test_disk_usage_endpoint(
         assert res.status_code == 200, res.json()
         disk_usage = res.json()  # currently: 6.38 Mio on Ubuntu.
         assert 6 * 1024 * 1024 < disk_usage < 7 * 1024 * 1024
+
+        # Create dummy outputs for the variant
+        outputs_dir = tmp_path / "internal_workspace" / variant_id / "outputs"
```

Comment:
Folder dir should be `variant_id / "output"` but it doesn't matter as it is a dummy test

Reply:
Well, it's a real mistake.